WcaOnRails/spec/models/registration_spec.rb

```diff
@@ -54,13 +54,13 @@
   end
 
   it "requires at least one event" do
-    registration.eventIds = ""
+    registration.registration_events.destroy_all
```

Comment:
So, I've just played with this a bit and discovered some interesting things.

Your suggestion doesn't work and there's a really simple reason for that: to remove the existing records we need to specify directly its `id` and include `_destroy: true` in these `registration_event_attributes`. So assigning just an empty Array doesn't work.

But I've just found out something cooler. We can achieve the same result with:
```ruby
registration.registration_events = []
```
Furthermore we can do things like: 
```ruby
registration.registration_events = [ RegistrationEvent.new(registration_id: registration.id, event_id: "333") ]
```
And it's saved immediately!
I think it's because when using `accept_nested_attributes_for` Rails sets `:autosave` to `true` for the relation.